src/main/java/jalau/at18/architects/StagesEphesus.java

```diff
@@ -0,0 +1,27 @@
+package jalau.at18.architects;
+
+enum StagesEphesus {
+    ONE(2, false, 3),
+    TWO(2, true, 3),
+    THREE(3, false, 4),
+    FOUR(3, true, 5),
+    FIVE(4, false, 7);
+
+    private int countOfResources;
+    private boolean equals;
```

Comment:
`countOfResources` and `points` are easy to understand, however, I'm not fully clear on `equals`, what does this attribute means?